packages/oui-select-picker/src/select-picker.controller.js

```diff
@@ -38,6 +38,12 @@ export default class SelectPickerController {
             }
         }
 
+        this.$scope.$watch("$ctrl.model", (newValue) => {
```

Comment:
This cannot be made through `$onChanges` hook ?

Reply:
No it doesn't work.

Reply:
It's because of the binding of the `model` (=) :)

Can you prefer using a function instead ?
```suggestion
        this.$scope.$watch(() => this.model, (newValue) => {
```